tests/jobs/test_scheduler_job.py

```diff
@@ -3328,7 +3327,7 @@ def test_reset_orphaned_tasks_external_triggered_dag(self):
         session.commit()
 
         reset_tis = scheduler.reset_state_for_orphaned_tasks(session=session)
-        self.assertEqual(1, len(reset_tis))
+        self.assertEqual(1, reset_tis)
```

Comment:
due to the same reason mentioned above, may be good to change the variable name to something like `num_reset_tis` to avoid confusion